### PR TITLE
Fixes consulars not having sensors

### PR DESCRIPTION
**Pull request by alsoandanswer**
Title.

### CODE REVIEW

**Comment by alsoandanswer** [comment]
!bugfix

**Comment by alsoandanswer** [comment]
!review

File: code/modules/clothing/under/miscellaneous.dm [hunk 1]
```diff
@@ -8,6 +8,7 @@
 /obj/item/clothing/under/gearharness
 	name = "gear harness"
 	desc = "Tight fitting gear harness."
+	has_sensor = SUIT_HAS_SENSORS
```
Comment (DreamySkrell) [inline code/modules/clothing/under/miscellaneous.dm:11]: anyone can take this, not just consulars

File: code/modules/clothing/under/miscellaneous.dm [hunk 2]
```diff
@@ -365,13 +366,15 @@
...
 	desc = "A navy suit and red tie, intended for the station's finest."
+	has_sensor = SUIT_HAS_SENSORS
 	icon_state = "navy_suit"
 	item_state = "bl_suit"
 	worn_state = "navy_suit"
 
 /obj/item/clothing/under/suit_jacket/burgundy
 	name = "burgundy suit"
 	desc = "A burgundy suit and black tie. Somewhat formal."
+	has_sensor = SUIT_HAS_SENSORS
```
Comment (DreamySkrell) [inline code/modules/clothing/under/miscellaneous.dm:377]: same here

**Review by DreamySkrell — CHANGES_REQUESTED** [review]
These two are generic items that anyone can take, and shouldn't have suit sensors (if only uniforms are supposed to have them).

I am also against `thing/consular` subtypes where the only difference from their parent `thing` type, is that they have sensors.

I do not think consulars should really have suit sensors in the first place - they're not SCC crew, they are guests here. They are not wearing SCC or corporate uniforms, but they just bring their clothes from their parent nation.

**Comment by lavillastrangiato** [comment]
> I do not think consulars should really have suit sensors in the first place - they're not SCC crew, they are guests here. They are not wearing SCC or corporate uniforms, but they just bring their clothes from their parent nation.
They are diplomats to the SCC's flagship. It would reflect very poorly on the SCC if a reason that diplomatic consul died was because they were not allowed to have vitals tracking, for some reason.

**Comment by DreamySkrell** [comment]
> > I do not think consulars should really have suit sensors in the first place - they're not SCC crew, they are guests here. They are not wearing SCC or corporate uniforms, but they just bring their clothes from their parent nation.
> 
> They are diplomats to the SCC's flagship. It would reflect very poorly on the SCC if a reason that diplomatic consul died was because they were not allowed to have vitals tracking, for some reason.

Also, not all consulars even really have uniforms, or makes sense for them to have uniforms - like Biesel consulars for example. If you want all consulars to have sensors, it should be via a system that is different from clothes.

**Comment by NonQueueingMatt** [comment]
#17898 will be merged instead

**Pull request closed without merging**